coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -88,7 +88,11 @@ def convert_nodes(
         except Exception as e:
             scope_names = node.get_scope_info()[0]
             op_location = '/'.join(scope_names[:-1])
-            logger.error(f"\n\nERROR - converting '{node.kind}' op (located at: '{op_location}'):\n")
+            if op_location == '':
```

Comment:
The `scope_names[-1]` could still have an useful information about the name of the top level op,
are we considering also including it in?

Reply:
Good catch. I've fixed that.